07.bitstream.semantics.md

```diff
@@ -1488,8 +1488,7 @@ When this function is called, the set frame refs
 process specified in [section 7.8][] is invoked.
 
 **ref_frame_idx[ i ]** specifies which reference frames are used by inter frames. It
-is a requirement of bitstream conformance that RefValid[ ref_frame_idx[ i ] ] is equal
-to 1, and that the selected reference frames match the current frame in bit depth,
+is a requirement of bitstream conformance that the selected reference frames match the current frame in bit depth,
 profile, chroma subsampling, and color space.
```

Comment:
I don't think this is right.  This loses the check on reference frames when order hints are being used (another way that RefValid can be cleared).